crates/release_plz_core/src/release.rs

```diff
@@ -131,9 +140,41 @@ fn release_package(
         info!("published {}", package.name);
     }
 
+    if input.gh_release {
+        publish_release(git_tag(&package.name, &package.version.to_string()), input);
```

Comment:
```suggestion
        publish_release(git_tag(&package.name, &package.version.to_string()), input).await;
```
You need to `await` this function because it's async.
Since this is the first await of the `release_package` function, you need to change the signature of the `release_package` function and add the `async` keyword, too.